### ISSUE TITLE
Tweak GoogLeNet and Inception family to match the torchvision implementations 

### ISSUE THREAD

**Issue by theabhirath** [FluxML/Metalhead.jl#196]
These changes will make it easier to port over pretrained weights for the models from PyTorch.

- [x] Right now, GoogLeNet matches the implementation in the paper, which does not use batch normalisation. However, torchvision uses [batch normalisation layers after the convolution layers](https://github.com/pytorch/vision/blob/6db1569c89094cf23f3bc41f79275c45e9fcb3f3/torchvision/models/googlenet.py#L285-L295) in both the inception block and the stem. It also uses `bias = false` for the convolution layer and `eps = 0.001` for the batch normalisation layer. We could decide if similar to VGG, we can have a toggle for the batch normalisation in the model. (closed by #205)
- [x] InceptionV3, InceptionV4 and InceptionResNetv2 have the right placement of normalisation layers but similar to GoogLeNet, use [`bias = false` for the convolution layer and `eps = 0.001` for the batch normalisation layer.](https://github.com/pytorch/vision/blob/96aa3d928c6faaf39defd846966421679244412d/torchvision/models/inception.py#L398-L407) (should be closed by #198)

**Comment by darsnack** [comment]
A toggle would be a good idea so that the paper variant is still easily available.

**Comment by pri1311** [comment]
Hi, this issue seems like a good place to start contributing.
`
function convolution(kernel_size, inplanes, outplanes, batchNorm; kwargs...)
    if batchNorm
        return basic_conv_bn(kernel_size, inplanes, outplanes; kwargs...)
    else 
        return Conv(kernel_size, inplanes => outplanes; kwargs...)
    end
end
`
would implementing such a function and the calling it like this
`convolution((7, 7), inchannels, 64, batchNorm; stride = 2, pad = 3, bias = bias`
be a good way to refactor code for the particular issue?
Or do you have something else in mind?

**Comment by darsnack** [comment]
We already have support for turning off the batch norm [here](https://github.com/FluxML/Metalhead.jl/blob/88bcacb81b643066adecdc9bb108a6dbcaad9e3c/src/layers/conv.jl#L15-L17). The remaining task on this issue it to update the code for GoogLeNet to use that functionality.

**Comment by pri1311** [comment]
Aahh got it! thanks.
